zrml/court/src/lib.rs

```diff
@@ -187,315 +169,1166 @@ mod pallet {
         type WeightInfo: WeightInfoZeitgeist;
     }
 
-    #[pallet::error]
-    pub enum Error<T> {
-        /// It is not possible to insert a Juror that is already stored
-        JurorAlreadyExists,
-        /// An account id does not exist on the jurors storage.
-        JurorDoesNotExists,
-        /// On dispute or resolution, someone tried to pass a non-court market type
-        MarketDoesNotHaveCourtMechanism,
-        /// No-one voted on an outcome to resolve a market
-        NoVotes,
-        /// Forbids voting of unknown accounts
-        OnlyJurorsCanVote,
-    }
+    // Number of jurors for an initial market dispute
+    const INITIAL_JURORS_NUM: usize = 5;
+    /// The current storage version.
+    const STORAGE_VERSION: StorageVersion = StorageVersion::new(2);
+    // Weight used to increase the number of jurors for subsequent appeals
+    // of the same market
+    const SUBSEQUENT_JURORS_FACTOR: usize = 2;
```

Comment:
The good part of not using config parameters here is that we really put thought into the whole formula. The court system is pretty expensive storage wise. The point of using config parameters is the ability of pallet integrators to tweak the pallet. Although I don't see anyone to use this pre-defined pallet other than ourselves.

But I see your point. It's just that there are many ways to get the same functionality. It might be a better style to use the current approach, because we know the current limitations of the court system. But style is subjective..

Reply:
06af2fbfeac6fe208b8ae9ce408ab77a8480172d